.claude-workflow.yml

```
@@ -0,0 +1,159 @@
+  framework: "none"
+  description: "Rust-based image compression tool"
+
+# 分析配置 - 针对 Rust 项目优化
+analysis:
+  basics:
+    code_quality: true        # 代码质量检查
+    security_scan: true       # 安全扫描
+    dependency_check: true    # 依赖检查
+    test_coverage: true       # 测试覆盖率
+    documentation_check: true  # Rust 文档检查
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)